bin/foreman-tasks

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
```

Comment:
should it be called foreman-tasks ? wont that be confusing? (or conflicting in packaging)?